src/sourmash/search.py

```diff
@@ -235,16 +239,20 @@ def search_databases_with_abund_query(query, databases, **kwargs):
                               query=query,
                               query_filename=query.filename,
                               query_name=query.name,
-                              query_md5=query.md5sum()[:8]
+                              query_md5=query.md5sum()[:8],
+                              ksize=query.minhash.ksize,
         ))
     return x
 
 ###
 ### gather code
 ###
 
-GatherResult = namedtuple('GatherResult',
-                          'intersect_bp, f_orig_query, f_match, f_unique_to_query, f_unique_weighted, average_abund, median_abund, std_abund, filename, name, md5, match, f_match_orig, unique_intersect_bp, gather_result_rank, remaining_bp, query_filename, query_name, query_md5, query_bp')
+GatherResult = namedtuple('GatherResult', ['intersect_bp', 'f_orig_query', 'f_match', 'f_unique_to_query',
+                            'f_unique_weighted','average_abund', 'median_abund', 'std_abund', 'filename',
+                            'name', 'md5', 'match', 'f_match_orig', 'unique_intersect_bp', 'gather_result_rank',
+                            'remaining_bp', 'query_filename', 'query_name', 'query_md5', 'query_bp', 'ksize',
+                            'moltype', 'num', 'scaled', 'query_nhashes', 'query_abundance'])
```

Comment:
suggest `query_n_hashes`, to match what I'm using in sqlite3 databases and elsewhere.

Do we want to add `match_n_hashes` too?

Reply:
changed in https://github.com/sourmash-bio/sourmash/pull/1952/commits/644a4dd83eb7f6f48ad11c54122ddeecb35a9faa! Will save adding `match_n_hashes` for #1955, where I've already added it :)